doc/files.md

```diff
@@ -16,7 +16,7 @@
 
   - [Berkeley DB database based wallets](#berkeley-db-database-based-wallets)
 
-- [Notes](#notes)
+- [Installed Files](#installed-files)
```

Comment:
nit: if you're already editing this file consider fixing line 86:
> ... double-spends due to *the* lack of synchronization

Reply:
re: https://github.com/bitcoin/bitcoin/pull/31679#discussion_r2185083314

> nit: if you're already editing this file consider fixing line 86:
> 
> > ... double-spends due to _the_ lack of synchronization

Thanks, added commit to fix